handout/a11y/key-concerns/keyboard-accessibility.md

```diff
@@ -0,0 +1,61 @@
+# Keyboard Accessibility
+
+Keyboard accessibility is the ability of your application to be interacted with using just a keyboard. The more streamlined the site can be used this way, the more keyboard accessible it is. Keyboard accessibility is one of the largest aspects of web accessibility since it targets:
+
+- those with motor disabilities who can't use a mouse
+- blind users who typically prefer to navigate websites with their keyboard
+- those who prefer not to use a mouse.
+
+
+## Focus
+
+Keyboard interaction is driven by something called *focus*. In web applications, only one element on a document has focus at a time, and keypresses are done using that element as the main point of context. This element can be accessed programmatically through the `document.activeElement` DOM method and visually, this is often represented by a glowing border around the element.
+
+![Focus border](./focus-border.png)
```

Comment:
I have seen many people removing focus borders altogether because they find them ugly, so It may be worth mentioning that focus styling + outlines can be explicitly added.